src/gluonts/model/simple_feedforward/_network.py

```diff
@@ -92,34 +90,31 @@ def __init__(
             )
             self.scaler = MeanScaler() if mean_scaling else NOPScaler()
 
-    def get_distr(self, F, past_target: Tensor) -> Distribution:
+    def get_distr_args(self, F, past_target: Tensor):
         """
         Given past target values, applies the feed-forward network and
-        maps the output to a probability distribution for future observations.
-
+        maps the output to the parameter of probability distribution for future observations.
         Parameters
         ----------
         F
         past_target
             Tensor containing past target observations.
             Shape: (batch_size, context_length, target_dim).
-
         Returns
         -------
-        Distribution
-            The predicted probability distribution for future observations.
+        distr_args: the parameters of distribution
+        loc: an array of zeros with the same shape of scale
+        scale: 
+        
```

Comment:
These should be adjusted to follow the same layout as other methods (see hybrid_forward in other classes here).

The layout is:

```
<type>
    <description>
```

for example

```
Tuple[Tensor, Tensor Tensor]
    A tuple of three tensors ``distr_args``, ``loc``, and ``scale``: the first contains the output
    distribution parameters, and the others are location and scale of the distribution.
```